tests/__init__.py

```diff
@@ -0,0 +1,147 @@
+                                  r'F:\U24\workflow_ephys_data\subject2\session1',
+                                  r'F:\U24\workflow_ephys_data\subject2\session2',
+                                  r'F:\U24\workflow_ephys_data\subject3\session1',
+                                  r'F:\U24\workflow_ephys_data\subject4\experiment1',
+                                  r'F:\U24\workflow_ephys_data\subject5\2018-07-03_19-10-39']
+
+    sessions_csv_fp = pathlib.Path('./tests/user_data/sessions.csv')
+
+    input_sessions.to_csv(sessions_csv_fp)  # write csv file
+
+    yield input_sessions
+
+    sessions_csv_fp.unlink()  # delete csv file after use
+
+
+@pytest.fixture
+def testdata_paths():
+    return {
+        'npx3A-p0-ks': 'subject5/2018-07-03_19-10-39/probe_1/ks2.1_01',
+        'npx3A-p1-ks': 'subject5/2018-07-03_19-10-39/probe_1/ks2.1_01',
+        'oe_npx3B-ks': 'subject4/experiment1/recording1/continuous/Neuropix-PXI-100.0/ks',
+        'npx3A-p0': 'subject5/2018-07-03_19-10-39/probe_1',
+        'oe_npx3B': 'subject4/experiment1/recording1/continuous/Neuropix-PXI-100.0',
+    }
+
+
+@pytest.fixture
+def kilosort_paramset():
+    from workflow_ephys.pipeline import ephys
+
+    params_ks = {
+        "fs": 30000,
+        "fshigh": 150,
+        "minfr_goodchannels": 0.1,
+        "Th": [10, 4],
+        "lam": 10,
+        "AUCsplit": 0.9,
+        "minFR": 0.02,
+        "momentum": [20, 400],
+        "sigmaMask": 30,
+        "ThPr": 8,
+        "spkTh": -6,
+        "reorder": 1,
+        "nskip": 25,
+        "GPU": 1,
+        "Nfilt": 1024,
+        "nfilt_factor": 4,
+        "ntbuff": 64,
+        "whiteningRange": 32,
+        "nSkipCov": 25,
+        "scaleproc": 200,
+        "nPCs": 3,
+        "useRAM": 0
+    }
+
+    # doing the insert here as well, since most of the test will require this paramset inserted
+    ephys.ClusteringParamSet.insert_new_params(
+        'kilosort2', 0, 'Spike sorting using Kilosort2', params_ks)
+
+    yield params_ks
+
+    with dj.config(safemode=False):
+        ephys.ClusteringParamSet.delete()
+
+
+@pytest.fixture
+def clustering_tasks(kilosort_paramset):
+    from workflow_ephys.pipeline import ephys, get_ephys_root_data_dir
+
+    root_dir = pathlib.Path(get_ephys_root_data_dir())
+    for ephys_rec_key in (ephys.EphysRecording - ephys.ClusteringTask).fetch('KEY'):
+        ephys_file = root_dir / (ephys.EphysRecording.EphysFile & ephys_rec_key).fetch('file_path')[0]
+        recording_dir = ephys_file.parent
+        kilosort_dir = next(recording_dir.rglob('spike_times.npy')).parent
+        ephys.ClusteringTask.insert1({**ephys_rec_key,
+                                      'paramset_idx': 0,
+                                      'clustering_output_dir': kilosort_dir.as_posix()})
+
+    ephys.Clustering.populate()
+
+    yield
+
+    with dj.config(safemode=False):
+        ephys.ClusteringTask.delete()
```

Comment:
This deletes the content of this table but leaves behind the actual table, schema, and other tables + data involved in dependents. The proper testing procedure is to teardown anything that was generated within this fixture. If importing created the schema then it should be torndown after yield.